js/ui/codeView.js

```diff
@@ -360,6 +360,11 @@ var WindowTrackingButton = new Lang.Class({
     }
 });
 
+function _constructLoadFlatpakValue(app, appManifest) {
```

Comment:
The function was renamed (now it has an underscore in front) in this commit, but the call site was not updated to use the new name.

Reply:
Fixed.